spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/SpatialDilatedConvolution.scala

```diff
@@ -61,7 +62,9 @@ class SpatialDilatedConvolution[T: ClassTag](
   val padH: Int = 0,
   val dilationW: Int = 1,
   val dilationH: Int = 1,
-  private var initMethod: InitializationMethod = Default
+  private var initMethod: InitializationMethod = Default,
+  val wRegularizer: Regularizer[T] = null,
```

Comment:
The constructor list is too long. Can we move the new constructor parameter out?